main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # 1. Create:
+    genres_for_create = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    actors_for_create = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
```

Comment:
The list `actors_for_create` is a list of tuples which is a good practice according to the checklist (#2). However, the list `genres_for_create` is a list of strings. Consider changing it to a list of tuples for consistency and better iterating.